pkgs/applications/audio/youtube-music/default.nix

```diff
@@ -51,17 +46,15 @@ stdenv.mkDerivation (finalAttrs: {
     dontBuild = true;
     dontFixup = true;
     outputHashMode = "recursive";
-    outputHashAlgo = "sha256";
     outputHash = {
-      x86_64-linux = "sha256-Oy11V7FXfVhLUR9gX0sjQEFuVPFpbaVdT518oOSLcvA=";
-      aarch64-linux = "sha256-6nXemaGiQjp2stjjKItPJ62VcH5Q5pRf63qKtl2haXI=";
-      x86_64-darwin = "sha256-jSMAw+AMD63vqPckZjblw4EDngA4E8h0WlsZu3hUShY=";
+      x86_64-linux = "sha256-SOtueFJ+w6k9BQMbyd7xtbQ4CusExIEO5gMRywu4+0I=";
+      aarch64-linux = "sha256-MHnVoQMqDvCqNpmV/lFKGlGJErXZEKPjdWfgEUszu90=";
+      x86_64-darwin = "sha256-XK7qof8HQ3qf7Dwql04+0yWvHOPI5YanfID+y1VuM+Y=";
       aarch64-darwin = "sha256-zujXURpIcw7IOw63AW167h6cywYXydhHZMzA2apGZAs=";
```

Comment:
If someone could provide that it would be awesome. ofborg is taking it's sweet time

Reply:
```suggestion
      aarch64-darwin = "sha256-VrQdKrji/U3xjRfav87voUEisewukVRB4zAUBW4HOIw=";
```